src/Main.java

```diff
@@ -1,5 +1,86 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задание 1");
+        int salary = 15000;
+        int total = 0;
+        int i = 0;
+        while (total <= 2_459_000) {
+            total = total + salary;
+            i = i + 1;
+            System.out.println("Месяц " + i + " сумма накоплений равна " + total + " рублей ");
+        }
+        System.out.println("Задача 2");
+        int g = 0;
+        while (g < 10) {
+            g = g + 1;
+            System.out.print(g + " ");
+        }
+        System.out.println();
+        int h = 11;
+        for (; h > 1; ) {
+            h = h - 1;
+            System.out.print(h + " ");
+        }
+        System.out.println();
+
+        System.out.println("Задание 3");
+        int people = 12_000_000;
+        int difference = people / 1000;
+        int born = 17;
+        int death = 8;
+        for (int year = 1; year <= 10; year++) {
+            people = people + (difference * (born - death));
+            System.out.println("Год " + year + " численность населения составляет " + people);
+        }
+        System.out.println("Задание 4");
+        int cash = 15000;
+        int zero = 0;
+        int r = 0;
+        while (cash < 12_000_000) {
+            r = r + 1;
+            zero = zero + cash;
+            zero = zero / 100;
+            zero = zero * 7;
+            cash = cash + zero;
+            System.out.println("Месяц " + r + " сумма накоплений равна " + cash + " рублей ");
+        }
+        System.out.println("Задание 5");
+        int cash1 = 15000;
+        int zero1 = 0;
+        int r1 = 0;
+        while (zero1 < 12_000_000) {
+            zero1 = zero1 + (zero1 / 100) * 7;
+            zero1 = zero1 + cash1;
+            r1++;
+            if (r1 % 6 == 0) {
+                System.out.println("Месяц " + r1 + " сумма накоплений равна " + zero1 + " рублей ");
+            }
+        }
+        System.out.println("Задание 6");
+        int cash2 = 15000;
+        int zero2 = 0;
+        int r2 = 0;
+         while (r2 < 108) {
```

Comment:
108 - непонятно. Понятнее 9*12
```suggestion
         while (r2 < 9*12) {
```